js/src/forum/states/PostStreamState.js

```diff
@@ -172,7 +172,7 @@ class PostStreamState {
    * @return {Promise}
    */
   loadNearIndex(index) {
-    if (index >= this.visibleStart && index <= this.visibleEnd) {
```

Comment:
This was already done in https://github.com/flarum/core/commit/09e2736cbcc267594b660beabbd001d9030f9880, so a rebase would be better than repeating this change